Cargo.toml

```diff
@@ -12,12 +12,14 @@ nightly = ["embedded-io?/async", "embedded-svc?/nightly", "embedded-svc?/experim
 notification = ["futures-util", "embassy-sync"]
 edge-net = ["nightly", "dep:edge-net", "embassy-futures", "embedded-io", "log", "serde", "heapless", "postcard"]
 embedded-svc = ["nightly", "dep:embedded-svc", "embassy-futures", "log", "serde", "heapless", "postcard"]
+embedded-svc-prost = ["nightly", "dep:embedded-svc", "embassy-futures", "log", "prost", "heapless", "postcard"]
```

Comment:
The support for `prost` should be implemented for `embedded-svc` (as you did) *but also* for the `edge-net` feature as well.  It is just swapping out the `postcard` serde in the `ws` module with `prost`.

Therefore, best is to just have a `prost` feature, which in fact you can implicitly do by marking the `prost` dependency as optional.